integration_tests/model_visualization_test.py

```diff
@@ -1,30 +1,45 @@
+from pathlib import Path
+
 import keras
 from keras.src.utils import plot_model
 
 
-def plot_sequential_model():
+def assert_file_exists(path):
+    assert Path(path).is_file(), "File does not exist"
+
+
+def test_plot_sequential_model():
     model = keras.Sequential(
         [
             keras.Input((3,)),
             keras.layers.Dense(4, activation="relu"),
             keras.layers.Dense(1, activation="sigmoid"),
         ]
     )
+
     plot_model(model, "sequential.png")
+    assert_file_exists("sequential.png")
+
     plot_model(model, "sequential-show_shapes.png", show_shapes=True)
+    assert_file_exists("sequential-show_shapes.png")
```

Comment:
Can you put filenames in a variable so they're not duplicated?

Reply:
Yes, I updated the source code as your advice.